chef/cookbooks/ohai/files/default/plugins/linux/crowbar_libvirt.rb

```diff
@@ -0,0 +1,23 @@
+provides "crowbar_ohai"
+
+require_plugin "kernel"
+require_plugin "dmi"
+require_plugin "linux::s390x"
+
+libvirt_uuid = nil
+
+if kernel[:machine] == "s390x"
+  if s390x[:system][:manufacturer] == "KVM"
+    libvirt_uuid = s390x[:system][:uuid]
+  end
+else
+  manufacturer = dmi[:system] ? dmi[:system][:manufacturer] : "unknown"
+  if ["Bochs", "QEMU"].include? manufacturer
+    libvirt_uuid = dmi[:system][:uuid]
+  end
+end
+
+unless libvirt_uuid.nil?
```

Comment:
Isn't it simpler to remove the condition? This will result in the guest_uuid attribute being nil when we're not virtualized, but that'd be reasonable, and would avoid checks to see if crowbar_ohai[:libvirt] is nil or not in other locations.

(no need to change if you're happy the current way)